<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>patient_agent</class>
 <widget class="QDialog" name="patient_agent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>801</width>
     <height>551</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ID_PATIENT:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NOM_PATIENT:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PRENOM_PATIENT:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>120</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>DATE_PATIENT:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TEL_PATIENT:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ADRESSE:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prenom">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_date">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_tel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_adresse">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>360</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>511</width>
       <height>351</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../img/16379.jpg</pixmap>
     </property>
    </widget>
    <zorder>label_14</zorder>
    <zorder>label</zorder>
    <zorder>lineEdit_id</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>lineEdit_nom</zorder>
    <zorder>lineEdit_prenom</zorder>
    <zorder>lineEdit_date</zorder>
    <zorder>lineEdit_tel</zorder>
    <zorder>lineEdit_adresse</zorder>
    <zorder>pushButton</zorder>
    <zorder>pushButton_4</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Modifier</string>
    </attribute>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Donner id patient:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NOM_PATIENT:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PRENOM_PATIENT:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>DATE_PATIENT:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TEL_PATIENT:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ADRESSE:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_13">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>501</width>
       <height>351</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../img/16379.jpg</pixmap>
     </property>
    </widget>
    <zorder>label_15</zorder>
    <zorder>label_8</zorder>
    <zorder>lineEdit_8</zorder>
    <zorder>label_9</zorder>
    <zorder>label_10</zorder>
    <zorder>label_11</zorder>
    <zorder>label_12</zorder>
    <zorder>label_13</zorder>
    <zorder>lineEdit_9</zorder>
    <zorder>lineEdit_10</zorder>
    <zorder>lineEdit_11</zorder>
    <zorder>lineEdit_12</zorder>
    <zorder>lineEdit_13</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>pushButton_5</zorder>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Supprimer</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Donner id patient:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>270</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>2</y>
       <width>481</width>
       <height>341</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../img/16379.jpg</pixmap>
     </property>
    </widget>
    <zorder>label_16</zorder>
    <zorder>label_7</zorder>
    <zorder>lineEdit_id_2</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>pushButton_6</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Afficher</string>
    </attribute>
    <widget class="QTableView" name="tablepatient">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>471</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>491</width>
       <height>341</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../img/16379.jpg</pixmap>
     </property>
    </widget>
    <zorder>label_17</zorder>
    <zorder>tablepatient</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
